docs/manual/workflow-steps/azure.md

```diff
@@ -0,0 +1,202 @@
+# Azure Job Steps
+
+Azure is Microsoft's web services product. It is a cloud computing platform that offers a variety of services including but not limited to storage, networking and analytics. Azure's integration with Rundeck allows you to edit and utilized your Azure services in a Rundeck job. 
+
+## Azure / Storage / Copy
+
+This job step allows users to copy files to and from their Azure storage. 
+
+![Azure - Copy](~@assets/img/azure_copy.png)
+
+- **Storage Account**
+: the name of the Azure storage account you wish to use
+
+- **Access Key**
+: the file containing access key used to authenticate your account
+
+- **Source**
+: the path to the file you want to copy. It can be either an Azure URI or a local path (azure://container/path/file.ext or file://some/path/file.ext)
+
+- **Destination**
+: the path to where you want the file to be copied to. It can be either an Azue URI or a local path (azure://container/path/file.ext or file://some/path/file.ext)
+
+## Azure / Storage / Delete
+
+![Azure - Delete](~@assets/img/azure_delete.png)
+
+- **Storage Account**
+: the Azure storage account you wish to use
+
+- **Access Key**
+: the file containing access key used to authenticate your account
+
+- **Container Name**
+: the name of the Azure container that the file you want to delete is in
+
+- **Blob Path**
+: the path to the file that you want to delete. If the file is in a subfolder include the whole path like 'path/file.ext'
+
+## Azure / Storage / List
+
+![Azure - List](~@assets/img/azure_list.png)
+
+- **Storage Account**
+: the Azure storage account you wish to use
+
+- **Access Key**
+: the file containing access key used to authenticate your account
+
+- **Container Name**
+: the name of the Azure container that the you want to see the contents of
+
+## Azure / VM / Create
```

Comment:
Similar to Storage let's highlight the overview of what is possible here to put the reader's mind into the context of what they are trying to do.